blaseball_mike/tables.py

```diff
@@ -0,0 +1,113 @@
+import enum
+
+
+class Weather(enum.IntEnum):
+    VOID = 0
+    SUNNY = 1
+    OVERCAST = 2
+    RAINY = 3
+    SANDSTORM = 4
+    SNOWY = 5
+    ACIDIC = 6
+    SOLAR_ECLIPSE = 7
+    GLITTER = 8
+    BLOODWIND = 9
+    PEANUTS = 10
+    BIRD = 11
+    FEEDBACK = 12
+    REVERB = 13
+
+
+class Blood(enum.IntEnum):
+    SINGLE_A = 0
+    TRIPLE_A = 1
+    DOUBLE_A = 2
+    ACID = 3
+    BASE = 4
+    OH = 5
+    OH_NO = 6
+    WATER = 7
+    ELECTRIC = 8
+    LOVE = 9
+    FIRE = 10
+    PSYCHIC = 11
+    GRASS = 12
+
+    def __str__(self):
```

Comment:
you don't actually need to do this, just set the enum value to the string and return the value

```
>>> import enum
>>> class Bluh(enum.Enum):
...     BLUH = "bluh"
...
>>> Bluh.BLUH.value
'bluh'
```

Reply:
These values are stored as integers in the models though, so any lookups via the integer value will fail in that case.

```
>>> ret = Bluh(1)
ValueError: 1 is not a valid Bluh
```